src/modules/BootSupportCheck.rb

```diff
@@ -191,6 +194,8 @@ def check_zipl_part
         return false
       end
 
```

Comment:
@aschnell just to clarify, the comment ("it would be better.... bla") is just for you. Not expected to be in the final code. :-)